### PR TITLE
status generator for workflow

### PR DESCRIPTION
**Pull request by rpatil1998**

### CODE REVIEW

File: pkg/status/status.go [hunk 1]
```diff
@@ -35,3 +35,8 @@ func GenerateBuildMsg(stage Stage, state State, msg string) string {
 	statusMsg := []string{string(stage), string(state), msg}
 	return strings.Join(statusMsg, "|")
 }
+
+func GenerateWorkflowStatusMsg(stepPosition int32, state State, errorCode string, msg string) string {
```
Comment (burnerlee) [inline pkg/status/status.go]: i don't think `string(stepPosition)` works as expected, use fmt.Sprintf instead

Comment (burnerlee) [inline pkg/status/status.go]: also change to `errorCode, msg string` no need to define types in 2 consecutive arguments of the same type

**Review by burnerlee — CHANGES_REQUESTED** [review]

File: pkg/status/status.go (hunk 1 above)
Comment (rpatil1998) [reply pkg/status/status.go]: not working either 
strconv.Itoa(int(stepPosition))

Comment (rpatil1998) [reply pkg/status/status.go]: ok

**Review by burnerlee — APPROVED** [review]

**Pull request merged**